frontend/pages/Tags.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import ParisMap from "../components/ParisMap";
+import LoadingPage from "./LoadingPage";
+import * as PropTypes from "prop-types";
+import PhotoViewer from "../components/PhotoViewer";
+
+class Tags extends PhotoViewer {
+    constructor(props) {
+        super(props);
+        this.state = {
+            loading: true,
+            tagData: null
+        };
+    }
+
+    async componentDidMount() {
+        try {
+            const response = await fetch(`/api/tag/${this.props.tagName}/`);
+            if (!response.ok) {
+                this.setState({loading: false});
+            } else {
+                const tagData = await response.json();
+                this.setState({
+                    tagData,
+                    loading: false
+                });
+            }
+        } catch (e) {
+            console.log(e);
+        }
+    }
+
+    formatTag(tagName) {
+        let array = tagName.split(" ");
+        for (let i = 0; i < array.length; i++) {
+            array[i] = array[i].charAt(0).toUpperCase() + array[i].slice(1);
```

Comment:
I believe you can just accomplish this with a CSS property:
`text-transform: capitalize;` Let's remove this logic